stac_fastapi/tests/api/test_api.py

```diff
@@ -132,11 +135,12 @@ async def test_app_fields_extension_query(app_client, ctx, txn_client):
         json={
             "query": {"proj:epsg": {"gte": item["properties"]["proj:epsg"]}},
             "collections": ["test-collection"],
+            "fields": {"include": ["properties.datetime", "properties.proj:epsg"]},
         },
     )
     assert resp.status_code == 200
     resp_json = resp.json()
-    assert list(resp_json["features"][0]["properties"]) == ["datetime", "proj:epsg"]
+    assert set(resp_json["features"][0]["properties"]) == set(["datetime", "proj:epsg"])
```

Comment:
Looks great. Why do you use set() here? Should it also be used in the `test_app_fields_extension` test above?

Reply:
The list came back in the wrong order when I was running the tests so I used set. It should probably be used in the other test too - you're right - and in the opensearch tests. I only had the error once.

Reply:
I guess it wouldn't be used in the above test because it is only expecting one item in the list.